client/src/app/services/word-list.actions.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core';
+import { NgRedux } from '@angular-redux/store';
+import { AppState } from '../store/app-state';
+import { WordListRestService } from './wordListRest.service';
+import { Word } from '../models/word';
+
+@Injectable()
+export class WordListActions {
+  static readonly REQUEST_WORD_LIST_SUCCESS = 'REQUEST_WORD_LIST_SUCCESS';
+  static readonly REQUEST_ADD_WORD = 'REQUEST_ADD_WORD';
+  static readonly REQUEST_UPDATE_WORD = 'REQUEST_UPDATE_WORD';
+  static readonly REQUEST_REMOVE_WORD = 'REQUEST_REMOVE_WORD';
+
+  constructor(private ngRedux: NgRedux<AppState>,
+              private wordListRestService: WordListRestService){
+
+  }
+
+  getWordList(){
+    this.wordListRestService.getData().subscribe(wordList => {
+      this.ngRedux.dispatch({
+        type: WordListActions.REQUEST_WORD_LIST_SUCCESS,
+        wordList
+      });
+    });
+  }
+
+  postData(word: Word) {
+    this.wordListRestService.postData(word).subscribe(word => {
+      this.ngRedux.dispatch({
+        type: WordListActions.REQUEST_ADD_WORD,
+        word
+      });
+    });
+  }
+
+  updateWord(word: Word){
+    this.wordListRestService.updateWord(word).subscribe(word => {
+      this.ngRedux.dispatch({
+        type: WordListActions.REQUEST_UPDATE_WORD,
+        word
+      });
+    });
+  }
+
+  removeWord(id: string){
+    this.wordListRestService.removeWord(id).subscribe(word => {
```

Comment:
No nie wiem jakie jest tutaj dobre podejście. Ja bym zrobił tak że przed `this.wordListRestService.removeWord()`, wykonałbym akcję `this.ngRedux.dispatch({type: WordListActions.REQUEST_WORD_LIST_LOAD});`. To daje nam możliwość np. zablokowania edycji słowa na sekunde zanim się zaktualizuje. 

Drugim podejściem może być to co mówisz. Zaktualizować słowo od razu, i jeżeli byłby błąd to wtedy wyrzucić jakieś powiadomienie. Bo w sumie ten request o zmiane wychodzi mega szybko ze stronki więc nie ma chyba specjalnie potrzeby informowania o ładowaniu. 

Ciężko wybrać które podejście lepsze :D